### ISSUE TITLE
commit messages suddenly changed from "oneline" format to showing several

### ISSUE THREAD

**Issue by smemsh** [integrations/slack#1205]
**Describe the bug**
Some time recently the commit messages in subscribed repos have changed.  They went from showing only the first line of the commit message, to showing several lines after the first one.  This has caused the previously succinct GitHub bot commit messages -- equivalent to `git log --oneline` and shown in neat rows one per line -- to now become fuller excerpts which look jumbled and confused, and are Too Much Information.

As a result of this change, our internal #commits channel suddenly got too noisy and hard to follow, with all the committers.  Our developer team already had rules for useful first lines, to contain enough information to understand the overview of the commit.  These new extra lines from the "details" portion of the commit message are extraneous in the context of the commit bot; the user already has a hyperlink available to see the full commit message.

Please bring the old way back, or at least make it configurable to get the first line only.  This is not something that should be unilaterally decided: some people will prefer more information, some will prefer only the first line.  (I would assume most people would prefer the old way, since that's how it was originally implemented...)

**To Reproduce**
Make a commit with several lines of text

**Expected behavior**
Only the first line is shown, like before.

**Comment by sushmita-m** [comment]
Thank you for raising this issue. We will change back the message to one line in the notification. The fix should be deployed tomorrow or early next week.
